.storybook/babel.config.js

```diff
@@ -0,0 +1,9 @@
+const presets = [
+    ['@babel/env', {
+        useBuiltIns: 'entry',
```

Comment:
This is the main difference between the storybook configuration and the build/release configuration: `entry` guarantees all core-js stuff is present / not resolved on a per-usage basis like with `usage`. For some reason in storybook per-usage does not identify what's needed correctly..